src/services/depositService.ts

```diff
@@ -141,6 +140,30 @@ export function createDepositEVMERC20Transaction(
   }
 }
 
+export function createDepositSiloToSiloTransaction(
+  assetAccountId: string,
+  userAddress: string,
+  amount: bigint,
+  depositAddress: string,
+  siloAddress: string
+): { to: Address; data: Hash } {
+  // TODO check chain type for userAddress as a rule, pass through the user chainType
+  const data = encodeFunctionData({
+    abi: siloAbi,
+    functionName: "safeFtTransferCallToNear",
+    args: [
+      assetAccountId as Address,
+      amount,
+      depositAddress,
+      userAddressToDefuseUserId(userAddress, ChainType.EVM),
```

Comment:
Please address these todo, it's important to not accidentally pass incorrect user id within the protocol.